lib/usb-hid-msc-lib.robot

```diff
@@ -16,7 +16,7 @@ Download ISO And Mount As USB
         Add USB To Qemu    img_name=${DL_CACHE_DIR}/${img_name}
     ELSE
         IF    "${DUT_CONNECTION_METHOD}" == "pikvm"
-            Upload Image To PiKVM    ${PIKVM_IP}    ${img_url}    ${img_name}
+            Upload Image To PiKVM    ${PIKVM_IP}    ${img_name}    ${img_url}
```

Comment:
`Upload Image PiKVM    ${pikvm_ip}    ${img_url}    ${img_name}    ${login}    ${password}` is the correct sequence

https://github.com/3mdeb/pikvm-rest-api/commit/529d434c1ac884f3cc5fa2e1a451829e95a83a0d
```
Upload Image To PiKVM
(...)
    [Arguments]    ${pikvm_ip}    ${img_url}    ${img_name}    ${login}=admin    ${password}=admin
    Upload Image PiKVM    ${pikvm_ip}    ${img_url}    ${img_name}    ${login}    ${password}
```
```
@keyword("Upload Image PiKVM")
def UploadImagePiKVM(
    pikvm_ip=str, img_url=str, img_name=str, login="admin", password="admin"
):
```

Reply:
@Psotas Are you requesting some change here? If yes, please just push the commit and link here.

Reply:
@Psotas I dropped this commit from this MR.